src/lib/paginator/index.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {NgModule} from '@angular/core';
+import {CommonModule} from '@angular/common';
+import {FormsModule} from '@angular/forms';
+import {MdCommonModule, OverlayModule} from '../core';
+import {MdButtonModule} from '../button/index';
+import {MdSelectModule} from '../select/index';
+import {MdPaginator} from './paginator';
+import {MdPaginatorIntl} from './paginator-intl';
+import {MdTooltipModule} from '../tooltip/index';
+
+
+@NgModule({
+  imports: [
+    CommonModule,
+    FormsModule,
+    MdButtonModule,
+    MdSelectModule,
+    MdTooltipModule,
+  ],
+  exports: [MdPaginator],
+  declarations: [MdPaginator],
+  providers: [MdPaginatorIntl],
```

Comment:
That's correct, it'll have multiple instances though that may be desired for the user if they want to customize their paginator not on internationalization but on the type of data. E.g. instead of "Items per page:" they can put "Users per page:".

@jelbourn what do you think